src/tests/conformance/mod.rs

```diff
@@ -1,29 +1,26 @@
-use std::net::{IpAddr, Ipv4Addr};
+use tempfile::TempDir;
 
 use crate::{
     protocol::codecs::binary::BinaryMessage,
-    setup::{config::ZIGGURAT_DIR, node::NodeBuilder},
+    setup::node::NodeBuilder,
     tools::{config::TestConfig, synth_node::SyntheticNode},
 };
 
 mod handshake;
 mod query;
+mod stateful;
 
 async fn perform_response_test(
     config: TestConfig,
     response_check: &dyn Fn(&BinaryMessage) -> bool,
 ) {
     // Build and start Ripple node
-    let mut node = NodeBuilder::new(
-        home::home_dir()
-            .expect("Can't find home directory")
-            .join(ZIGGURAT_DIR),
-        IpAddr::V4(Ipv4Addr::LOCALHOST),
-    )
-    .unwrap()
-    .build()
-    .await
-    .unwrap();
+    let temp = TempDir::new().expect("Unable to create TempDir");
```

Comment:
I think we should abstract this into the builder if possible: we don't actually need direct access to this struct in tests.

Reply:
We don't need direct access, but `TempDir` has special `drop` impl: it removes all content.

Reply:
As agreed via slack, we're leaving it as is for now.